furious/async.py

```diff
@@ -338,6 +340,12 @@ def start(self, transactional=False, async=False, rpc=None):
         try:
             ret = add(task, transactional=transactional)
         except taskqueue.TransientError:
+            if not retry_transient:
+                raise
+
+            import time
```

Comment:
OK - I'll move the time inline imports from this file and put them up top.  We've still got some other stdlib imports in this file, but I'll restrict my changes to the imports I've added.